src/main/resources/webapp/WEB-INF/views/data-man/webuserAdd.jsp

```diff
@@ -0,0 +1,72 @@
+<%--
+
+    SteVe - SteckdosenVerwaltung - https://github.com/steve-community/steve
+    Copyright (C) 2013-2024 SteVe Community Team
+    All Rights Reserved.
+
+    This program is free software: you can redistribute it and/or modify
+    it under the terms of the GNU General Public License as published by
+    the Free Software Foundation, either version 3 of the License, or
+    (at your option) any later version.
+
+    This program is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU General Public License for more details.
+
+    You should have received a copy of the GNU General Public License
+    along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+--%>
+<%@ include file="../00-header.jsp" %>
+<script type="text/javascript">
+    $(document).ready(function() {
+        <%@ include file="../snippets/datePicker-past.js" %>
+    });
+</script>
+<spring:hasBindErrors name="webuserForm">
+    <div class="error">
+        Error while trying to add a webuser:
+        <ul>
+            <c:forEach var="error" items="${errors.allErrors}">
+                <li>${error.defaultMessage}</li>
+            </c:forEach>
+        </ul>
+    </div>
+</spring:hasBindErrors>
+<c:if test="${webuserForm.pwerror == true}">
+        <div class="error">The password is to short or the password input is not identical.</div>
+</c:if>
+<div class="content"><div>
+<section><span>Add Webuser</span></section>
+    <form:form action="${ctxPath}/manager/webusers/add" modelAttribute="webuserForm">
+       <table class="userInput">
+            <thead><tr><th>New Webuser</th><th></th></thead>
+            <tbody>
+            <tr><td>Webusername:</td><td><form:input path="webusername" title="Web-Users Name"/></td></tr>
+            <tr><td>Password:</td><td><form:password path="password" title="Set the password"/></td></tr>
+            <tr><td>Password confirmation:</td><td><form:password  path="passwordComparison" title="Confirm the password"/></td></tr>
+            <tr><td>Roles:</td>
+                <td>
+                    <form><select id="myAuthoritiesList" name="authorities" path="authorities" title="List of roles/authoriies the web-user shall have.">
+                                    <option value="USER" >USER</option>
+                                    <option value="ADMIN" >ADMIN</option>
+                                    <option value="USER,ADMIN" >USER, ADMIN</option>
```

Comment:
you are deciding and hardcoding business values in view layer. maybe this should be an java enum?

Reply:
Admin is supposed to have more rights than user. What the purpose to have many roles for one user?

Reply:
> Admin is supposed to have more rights than user. What the purpose to have many roles for one user?

In general yes, but in some cases the user has privileges a admin does not. To keep this freedom for the future, I added the option that one web-user can have both roles.

Reply:
> in some cases the user has privileges a admin does not.

can you give examples?

i think the common practice is that an admin is a superset of user.

Reply:
hypothetical: the USER authority could be to manage the users (read, write of ocpp_tag, user) and the ADMIN authority manages only the charge-boxes. In this scenario there are some web-users which are allowed to do both.

Reply:
Based on what I understand your goal, I would like to propose implementing an additional user profile: CPO_USER (chargebox), EMP_USER (tag/user), and ADMIN (with full rights and privileges).

Reply:
see https://github.com/steve-community/steve/pull/1544#discussion_r1720797188

because of this, i think we should not try to find a golden solution. only 2 authorities (USER and ADMIN) will not be enough. the permission (what) distribution between roles (who) will be different for each org.

Reply:
If the division of read and write permissions between user and admin roles is not widely used, a straightforward approach would be to remove role-based access control in the first implementation and grant all users access to all privileges.

WDYT?

Reply:
I added a java enum, if some one don't want a combination of USER&ADMIN just out-comment it in the enum. But it work's if needed.